packages/tools/src/types/EventTypes.ts

```diff
@@ -201,6 +222,28 @@ type MouseDownEventDetail = NormalizedMouseEventDetail & {
   deltaPoints: IPoints;
 };
 
+/**
+ * EventDetail for touchstart event
+ */
+type TouchStartEventDetail = NormalizedTouchEventDetail & {
+  /** The starting points of the touch event. */
+  startPoints: ITouchPoints;
```

Comment:
Let the values that can be inheritted from MouseStartEventDetail be inheritted, and only add the extras as required.